app/assets/javascripts/application.js

```diff
@@ -12,8 +12,14 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require foundation
+//= require foundation.core.js.es6
+//= require foundation.util.triggers.js
+//= require foundation.util.mediaQuery.js
+//= require foundation.responsiveMenu.js
+//= require foundation.responsiveToggle.js
 //= require vue
 //= require_tree .
 
-$(function(){ $(document).foundation(); });
+$(function(){ 
+  $(document).foundation();
+});
```

Comment:
https://github.com/turbolinks/turbolinks/wiki/VueJs-and-Turbolinks looks like there is a way to get them to work together which would be nice and give us a bit of extra speed